frontend_app_file/src/container/PopupCreateFile.jsx

```diff
@@ -56,6 +59,43 @@ class PopupCreateFile extends React.Component {
     document.removeEventListener(CUSTOM_EVENT.APP_CUSTOM_EVENT_LISTENER, this.customEventReducer)
   }
 
+  componentDidUpdate (prevProps, prevState, snapshot) {
+    const { state, props } = this
+
+    if (state.uploadedFiles.length === state.uploadFiles.length && state.uploadStarted) {
```

Comment:
I think you should move all the code inside this condition into a separated function to keep a clean componentDidUpdate